packages/react-native/Libraries/AppDelegate/RCTRootViewFactory.mm

```diff
@@ -36,7 +36,7 @@
 #import <ReactCommon/RCTTurboModuleManager.h>
 #import <react/renderer/runtimescheduler/RuntimeScheduler.h>
 #import <react/renderer/runtimescheduler/RuntimeSchedulerCallInvoker.h>
-#import <react/runtime/JSRuntimeFactory.h>
```

Comment:
can we avoid this change? It would be great if we can avoid changing the import/include paths as this will make easier to land the change internally.

Reply:
Sure! I'm looking into this